node/rest/src/lib.rs

```diff
@@ -80,6 +83,10 @@ pub struct Rest<N: Network, C: ConsensusStorage<N>, R: Routing<N>> {
     handles: Arc<Mutex<Vec<JoinHandle<()>>>>,
     /// A reference to BlockSync,
     block_sync: Arc<BlockSync<N>>,
+    /// The number of ongoing deploy transaction verifications via REST.
+    num_verifying_deploys: Arc<AtomicUsize>,
```

Comment:
This is perfectly reasonable for this use case. In passing, another option that is slightly heavier but fairer, should that be desired, is a [`Semaphore`](https://docs.rs/tokio/latest/tokio/sync/struct.Semaphore.html).